src/include/txn.i

```diff
@@ -162,8 +162,10 @@ static inline void
 __txn_resolve_prepared_update(WT_SESSION_IMPL *session, WT_UPDATE *upd)
 {
     WT_TXN *txn;
+    WT_TXN_SHARED *txn_state;
```

Comment:
The local variable is called `txn_state`, should it be `txn_shared`?

Reply:
I don't mind, it's still a state. Happy to do another rename though.

Reply:
Ok, I have a commit ready for this.
I'm not going to push it just yet as it will blow up the diffs and make it hard to review.

Reply:
d1462b8